tests/functional/disaster-recovery/regional-dr/test_active_hub_down_and_restore.py

```diff
@@ -0,0 +1,193 @@
+import logging
+import time
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.pytest_customization.marks import tier4a, turquoise_squad
+from ocs_ci.framework import config
+from ocs_ci.ocs.acm.acm import validate_cluster_import
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.helpers.dr_helpers import (
+    failover,
+    relocate,
+    restore_backup,
+    create_backup_schedule,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    get_passive_acm_index,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    verify_drpolicy_cli,
+    verify_restore_is_completed,
+    get_scheduling_interval,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.utils import get_active_acm_index
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier4a
+@turquoise_squad
+class TestActiveHubDownAndRestore:
+    """
+    Test failover and relocate all apps when active hub down and restored RDR
+    """
+
+    def test_hub_recovery_failover_and_relocate(self, nodes_multicluster, dr_workload):
+
+        """
+        Tests to verify failover and relocate all apps when active hub down and restored RDR
+        """
+
+        # acm_obj = AcmAddClusters()
+        # Deploy Subscription and Appset based application
+        rdr_workload = dr_workload(
+            num_of_subscription=1, num_of_appset=1, switch_ctx=get_passive_acm_index()
+        )
+        logger.info(type(rdr_workload))
+        primary_cluster_name = get_current_primary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        secondary_cluster_name = get_current_secondary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        scheduling_interval = get_scheduling_interval(
+            rdr_workload[0].workload_namespace, rdr_workload[0].workload_type
+        )
+        # Create backup-schedule on active hub
+        create_backup_schedule()
```

Comment:
Backup-schedule creation should be part of hub recovery deployment/config. step IMO

Reply:
nope it's should be done by each tesst